strutil/cmdline.go

```diff
@@ -157,3 +158,29 @@ func KernelCommandLineSplit(s string) (out []string, err error) {
 	}
 	return out, nil
 }
+
+// GetKernelCommandLineKeyValue splits the given kernel command line string into
+// individual parameters using KernelCommandLineSplit, then matches for key
+// value pairs and returns the value of the specified key.
+func GetKernelCommandLineKeyValue(s string, key string) (string, error) {
+	params, err := KernelCommandLineSplit(strings.TrimSpace(s))
```

Comment:
I don't think I reviewed the PR that put KernelCommandLineSplit in strutil, it doesn't make much sense?  probably a good time to move these

Reply:
Samuele suggested to move the kernel cmdline handling to "osutil". I will push a tweak.